### PR TITLE
Replaced annotation with ingressClassName

### LINKED ISSUE #30126: Update ingress class setup note in the k8s.io/docs/concepts/services-networking/ingress-controllers/ page
Hello,

Kubernetes 1.18 introduced the IngressClass resource, effectively replacing the kubernetes.io/ingress.class annotation convention. The page above only advises to set ingress class as ingress.class annotation which is outdated. Update needed in all versions 1.18+ including  1.22.

Source:
https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation

### PR DESCRIPTION
**Pull request by fOO223Fr**
This PR resolves #30126
Updated content of https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/#using-multiple-ingress-controllers
with `ingressClassName` which replaces annotations

### CODE REVIEW

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -56,12 +56,10 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
 
 ## Using multiple Ingress controllers
 
-You may deploy [any number of ingress controllers](https://git.k8s.io/ingress-nginx/docs/user-guide/multiple-ingress.md#multiple-ingress-controllers) 
-within a cluster. When you create an ingress, you should annotate each ingress with the appropriate
-[`ingress.class`](https://git.k8s.io/ingress-gce/docs/faq/README.md#how-do-i-run-multiple-ingress-controllers-in-the-same-cluster) 
-to indicate which ingress controller should be used if more than one exists within your cluster.
+You may deploy any number of ingress controllers using [ingress class](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-class)
+within a cluster. Note the `.metadata.name` of your ingress class resource. When you create an ingress you would need that name to specify the `ingressClassName` field on your Ingress object (refer to [IngressSpec v1 reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.22/#ingressspec-v1-networking-k8s-io)). `ingressClassName` is a replacement of older [annotation method](https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation).
```
Comment (jihoon-seo) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:60]:
```suggestion
You may deploy any number of ingress controllers using [ingress class](/docs/concepts/services-networking/ingress/#ingress-class)
within a cluster. Note the `.metadata.name` of your ingress class resource. When you create an ingress you would need that name to specify the `ingressClassName` field on your Ingress object (refer to [IngressSpec v1 reference](/docs/reference/generated/kubernetes-api/v1.22/#ingressspec-v1-networking-k8s-io)). `ingressClassName` is a replacement of older [annotation method](/docs/concepts/services-networking/ingress/#deprecated-annotation).
```

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 2]
```diff
@@ -56,12 +56,10 @@ Kubernetes as a project supports and maintains [AWS](https://github.com/kubernet
...
 
-If you do not define a class, your cloud provider may use a default ingress controller.
+If you do not define a class name, your cloud provider may use a default ingress controller. [Default IngressClass](https://kubernetes.io/docs/concepts/services-networking/ingress/#default-ingress-class) can be set using the `ingressclass.kubernetes.io/is-default-class` annotation `true` on an `IngressClass` resource.
```
Comment (jihoon-seo) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:62]:
```suggestion
If you do not define a class name, your cloud provider may use a default ingress controller. [Default IngressClass](/docs/concepts/services-networking/ingress/#default-ingress-class) can be set using the `ingressclass.kubernetes.io/is-default-class` annotation `true` on an `IngressClass` resource.
```

**Review by jihoon-seo — COMMENTED** [review]
@fOO223Fr PTAL!

(Preview: https://deploy-preview-30171--kubernetes-io-main-staging.netlify.app/docs/concepts/services-networking/ingress-controllers/#using-multiple-ingress-controllers )

(This PR resolves #30126)

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:60]: Even better: link to the _new_ API reference page: [`/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec`](https://k8s.io/docs/reference/kubernetes-api/service-resources/ingress-v1/#IngressSpec).

I'd link to https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-ingress-class-deprecated for the deprecated annotation (we could add more text to that entry at the same time).

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 2 above)
Comment (sftim) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:62]:
```suggestion
If you do not specify an IngressClass for an Ingress, and your cluster has exactly one IngressClass marked as default, then Kubernetes [applies](/docs/concepts/services-networking/ingress/#default-ingress-class) the cluster's default IngressClass to the Ingress.
You mark an IngressClass as default by setting the [`ingressclass.kubernetes.io/is-default-class` annotation](/docs/reference/labels-annotations-taints/#ingressclass-kubernetes-io-is-default-class) on that IngressClass, with the string value `"true"`.
```

**Pull request merged**